testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -1302,8 +1302,8 @@ def test_get_velocities(self):
         v = self.ag.get_velocities()
         assert_(np.any(np.abs(v) > 1e-6), "velocities should be non-zero")
 
-    def test_vel_src(self):
-        assert_equal(self.universe.trajectory.ts.data['velocity_source'], 0)
+    # def test_vel_src(self):
```

Comment:
what is 'velocity_source' for ?. I didn't find anything in the code that used this meta-data field.